### DISCUSSION TITLE
大家都是怎么连接火币获取行情

### DISCUSSION THREAD

**Discussion by wdy199628** [veighna-global/vnpy_evo#17] [Q&A]
"key": "",
  "secret": "",
  "代理地址": "",
  "代理端口": "",

key和secret是火币网的公私钥吗，代理地址和端口又是怎么填的

**Comment by xldistance** [comment]
推荐使用v2rayn+trojan代理(自己买),代理地址"127.0.0.1",断开默认一般是"1080"也可以自己改
